include/aunit/reporters/aunit-reporter-trx.ads

```diff
@@ -0,0 +1,25 @@
+with Ada.Finalization;
```

Comment:
Again, bareboard runtimes do not have Ada.Finalization. You should either avoid using it or exclude corresponding reporter for all but full profile (you can see how this is done e.g. for File_IO).